electron/main.development.js

```diff
@@ -137,7 +137,7 @@ app.on('ready', async () => {
 
   try {
 
-    const ca = readCA(path.join(__dirname, '../tls/ca.crt'));
+    const ca = readCA(path.join(__dirname, '../../../tls/ca/ca.crt'));
```

Comment:
This change is breaking TLS configuration in development. With this configuration TLS configuration file needs to be outside of the project directory. Something to discuss tomorrow @domenkozar and @deepfire.

Reply:
Maybe we should put it in AppData as suggested by @domenkozar , where we have DB and logs